packages/action-menu/stories/action-menu.stories.ts

```diff
@@ -114,6 +159,14 @@ iconOnly.args = {
     visibleLabel: '',
 };
 
+export const tooltip_description_placement = (
```

Comment:
Convert these snake_case variables to camelCase to match the rest of the project.